### DISCUSSION TITLE
Explain Out of Memory

### DISCUSSION THREAD

**Discussion by Maxiwork** [Budibase/budibase#7663] [Help]
Hi everyone !

Can you explain me why that give me "Out of Memory" page please ?

![image](https://user-images.githubusercontent.com/111142619/188105037-67783c25-ca4e-4728-aca1-02c29adcbe9f.png)

Any solution to avoid this ?
Thank you !

**Comment by melohagan** [comment]
Hi @Maxiwork 

I have seen this myself a few times. It does appear to be being looked at: https://github.com/Budibase/budibase/issues/6594

In my case I was cloud hosted and had a number of REST queries on one screen. If you are self-hosted you may find better performance with the single image: https://docs.budibase.com/docs/docker#method-1---docker-run
